### ISSUE TITLE
Building instructions for Ubuntu don't work

### ISSUE THREAD

**Issue by mgaunard** [jatovm/jato#38]
I tried to follow the instructions to build on Ubuntu 14.04 but they don't work.
1) I wasn't able to build classpath at all, it's not finding a freetype header despite it being installed.
2) I had to add -lncurses to the build flags otherwise I had undefined references to ncurses symbols coming from /usr/lib/llvm-3.4/lib/libLLVMSupport.a

**Comment by penberg** [comment]
@mgaunard Which version of GNU Classpath did you attempt to build?

**Comment by penberg** [comment]
AFAICT, it's a LLVM problem on Ubuntu but as there's no harm in including ncurses, I fixed it up like that in commit b90be30d9e0a5be75cbaea1f3762ecc8ddafdbe8.

**Comment by mgaunard** [comment]
0.99, as instructed in the README

**Comment by penberg** [comment]
I think the problem is fixed in GNU Classpath master.

The following works for me on Ubuntu 14.04:

```
$ git clone git clone git://git.savannah.gnu.org/classpath.git
$ cd classpath
$ sh autogen.sh
$ ./configure --disable-Werror --disable-plugin
$ make -j8
$ sudo make install
```

I guess we should put that in the README instead of telling people to download 0.99.

**Comment by mgaunard** [comment]
Thanks, I can confirm that this works correctly. You might want to update the README to reflect this.

**Comment by penberg** [comment]
Fixed by #39. Closing
